policy/comCamMapper.yaml

```diff
@@ -26,122 +26,96 @@ exposures:
     inputOnly: true
   raw_amp:
     level: Amp
+    # NB If type is changed to an exposureI then constructDark breaks. If changing this be sure to test calibs
     persistable: DecoratedImageI
     python: lsst.afw.image.DecoratedImageI
     storage: FitsStorage
     tables: raw
-    template: raw/%(run)s/%(raft)s/%(ccd)s/%(raft)s-%(ccd)s-%(visit)06d.fits[%(channel)d]
+    template: raw/%(run)s/%(ccd)s/%(ccd)s-%(visit)06d.fits[%(channel)d]
   postISRCCD:
-    level: Ccd
-    persistable: ExposureF
-    python: lsst.afw.image.ExposureF
-    storage: FitsStorage
-    tables: raw
-    template: postISRCCD/postISRCCD_v%(visit)d_f%(filter)s.fits
+    template: 'postISRCCD/postISRCCD_v%(visit)d_%(ccd)s_f%(filter)s.fits'
   calexp:
-    level: Ccd
-    persistable: ExposureF
-    python: lsst.afw.image.ExposureF
-    storage: FitsStorage
-    tables: raw
-    template: calexp/calexp_v%(visit)d_f%(filter)s.fits
-  fcr:
-    level: Ccd
-    persistable: ExposureI
-    python: lsst.afw.image.ExposureI
-    storage: FitsStorage
+    template: calexp/calexp_v%(visit)d_%(ccd)s_f%(filter)s.fits
+  icExp:
+    template: icExp/icExp_v%(visit)d_%(ccd)s_f%(filter)s.fits
+  wcs:
     tables:
     - raw
     - raw_visit
-    template: fcr/v%(visit)d_f%(filter)s.fcr.fits
-  icExp:
-    level: Ccd
-    persistable: ExposureF
-    python: lsst.afw.image.ExposureF
-    storage: FitsStorage
-    tables: raw
-    template: icExp/icExp_v%(visit)d_f%(filter)s.fits
-  wcs:
+    template: wcs/v%(visit)d_%(ccd)s_f%(filter)s.wcs.fits
+  fcr:
     level: Ccd
     persistable: ExposureI
     python: lsst.afw.image.ExposureI
     storage: FitsStorage
     tables:
     - raw
     - raw_visit
-    template: wcs/v%(visit)d_f%(filter)s.wcs.fits
+    template: fcr/v%(visit)d_%(ccd)s_f%(filter)s.fcr.fits
 
 calibrations:
   bias:
-    columns: date
+    columns:
+    - date
     level: Ccd
     obsTimeName: date
-    persistable: DecoratedImageF
-    python: lsst.afw.image.DecoratedImageF
+    persistable: ExposureF
+    python: lsst.afw.image.ExposureF
```

Comment:
Are these entries needed for the cals? I would have expected obs_base to provide suitable defaults.

Reply:
I would expect so too, but couldn't find any. I agree, all these very standard calibration types should be moved to obs_base at some point.